packages/odie-client/src/context/index.tsx

```diff
@@ -65,6 +65,7 @@ export const OdieAssistantProvider: React.FC< OdieAssistantProviderProps > = ( {
 	extraContactOptions,
 	selectedSiteId,
 	selectedSiteURL,
+	initialMessage,
```

Comment:
I was confused by this, thinking it was the first message that the user would have seen. Can we change the naming to reflect that we're using this to send info to the Happiness Engineers?